.github/workflows/release.yml

```diff
@@ -282,6 +283,90 @@ jobs:
               with:
                   name: binaries
                   path: binaries
+    check-wasm-size:
+      name: "Check WASM runtimes with Twiggy"
+      runs-on: ubuntu-latest
+      if: github.event_name == 'pull_request'
+      needs: ["set-tags", "build"]
+      env:
+        GH_TOKEN: ${{ github.token }}
+        RUNTIMES_AS_STRING: 'dancebox_runtime,flashbox_runtime,dancelight_runtime,container_chain_template_simple_runtime,container_chain_template_frontier_runtime'
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+            ref: ${{ needs.set-tags.outputs.git_ref }}
+        # With rustup's nice new toml format, we just need to run rustup show to install the toolchain
+        # https://github.com/actions-rs/toolchain/issues/126#issuecomment-782989659
+        - name: Setup Rust toolchain
+          run: |
+            rustup override unset
+            rustup show
+        - name: Download Twiggy
+          run: cargo install twiggy
+        - name: Lookup for latest target branch build
+          run: |
+            TARGET_BRANCH=${{ github.event.pull_request.base.ref }}
+            LATEST_TARGET_BRANCH_BUILD=$(gh run -R moondance-labs/tanssi list -w CI --limit=100 --json databaseId,url,headBranch,event,status,conclusion,createdAt --jq ".[] | select(.headBranch == \"$TARGET_BRANCH\" and .event == \"push\" and .status == \"completed\" and .conclusion == \"success\") | .databaseId" | head -n 1)
+            echo "LATEST_TARGET_BRANCH_BUILD=$LATEST_TARGET_BRANCH_BUILD" >> $GITHUB_OUTPUT
+        - name: Download latest target branch build artifacts
+          run: |
+            gh run download $LATEST_TARGET_BRANCH_BUILD -n runtimes --dir runtimes-target-branch
+
+        - name: "Download branch built runtime"
+          uses: actions/download-artifact@v4
+          with:
+            name: runtimes
+            path: runtimes-current-branch
+        - name: Prepare file
+          run: |
+            # Create or truncate the file
+            echo "" > runtime_size_report.md
```

Comment:
If you are always overwriting the file. Is there any reason for appending?

Reply:
makes it easier to write "individual" messages